test/inspector/test-scriptparsed-context.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+const common = require('../common');
+common.skipIfInspectorDisabled();
+common.crashOnUnhandledRejection();
+const { NodeInstance } = require('./inspector-helper.js');
+const assert = require('assert');
+
+const script = `
+  'use strict';
+  const assert = require('assert');
+  const vm = require('vm');
+  const { kParsingContext } = process.binding('contextify');
+  debugger;
+
+  global.outer = true;
+  global.inner = false;
+  const context = vm.createContext({
+    outer: false,
+    inner: true
+  });
+  debugger;
+
+  const scriptMain = new vm.Script("outer");
+  debugger;
+
+  const scriptContext = new vm.Script("inner", {
+    [kParsingContext]: context
+  });
+  debugger;
+
+  assert.strictEqual(scriptMain.runInThisContext(), true);
+  assert.strictEqual(scriptMain.runInContext(context), false);
+  assert.strictEqual(scriptContext.runInThisContext(), false);
+  assert.strictEqual(scriptContext.runInContext(context), true);
+  debugger;
+
+  vm.runInContext('inner', context);
+  debugger;
+
+  vm.runInNewContext('Array', {});
+  debugger;
+`;
+
+async function getContext(session) {
+  const created =
+    await session.waitForNotification('Runtime.executionContextCreated');
```

Comment:
Teeny tiny style nit but line continuations are indented by four spaces.  (I don't know why the linter doesn't enforce that.)